requirements/base.in

```diff
@@ -1,3 +1,4 @@
+pip==20.3.1
```

Comment:
Please also remove this. Thanks

Reply:
Sorry, just saw your other comment, Why is it needed to be pinned?

Reply:
@hannesdiedrich when we compile the requirements, it is important to have it pinned in the .in files , this will be translated to the .txt files

Reply:
I see now, didnt remember that we pinned it to 20.3.1 everywhere I thought we always install the latest version now. OK then. leave it as is